src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -2,15 +2,16 @@
 
 import com.tvtracker.enterprise.dto.MediaEntry;
 import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 
-@Component
+@Repository
 public class MediaEntryDAOStub implements  IMediaEntryDAO {
-    HashMap<Integer, MediaEntry> entriesByID = new HashMap<Integer, MediaEntry>();
-    HashMap<String, HashMap<Integer, MediaEntry>> entriesByUsername = new HashMap<String, HashMap<Integer, MediaEntry>>();
+    HashMap<Integer, MediaEntry> entriesByID = new HashMap<>();
+    HashMap<String, HashMap<Integer, MediaEntry>> entriesByUsername = new HashMap<>();
```

Comment:
I did this here as well as a few other places its what I've been told is syntactic sugar it basically just cleans up the code because you've already declared what types are going to be in the hashmap. 
https://www.baeldung.com/java-clean-code

Reply:
We used to have to add generics on the constructor side as well, but that ended in Java 1.8.  Now, you can just use the "diamond operator", as you point out here.

Sometimes I add generics on the constructor side, just out of habit...